tests/resources/MST_mirror_2f_measurements.schema.yml

```diff
@@ -14,15 +14,15 @@ base_schema_version: 0.1.0
 data:
   - type: datatable
     table_columns:
-      - name: mirror_panel_id
+      - name: mirror_id
```

Comment:
Same comment as above, this time tagging @GernotMaier. 
Why did you go from `mirror_panel_id` to `mirror_id`? I think we wanted to keep always the reference to panel when dealing with mirror panels and not full telescope mirrors.

Reply:
I completely agree that it should be mirror_panel_id. Not sure why this was changed.